src/main/kotlin/se/uulm/snowballr/backend/repository/ProjectTableRepo.kt

```diff
@@ -37,6 +39,23 @@ interface IProjectTableRepo {
      * Returns all active projects stored in the database.
      */
     suspend fun getAllProjects(): List<Project>
+
+    /**
+     * Retrieves a list of projects from the database in which the user with the specified [userId] is a member.
+     *
+     * @param userId The unique identifier of the user whose associated projects are to be fetched.
+     * @param statusFilter (optional) Filter to specify which project status the fetched projects should have.
+     *        By default, [ProjectStatus.PROJECT_STATUS_ACTIVE] is used, which includes both active
+     *        and active-locked projects.
+     *        If set to another value (e.g., [ProjectStatus.PROJECT_STATUS_DELETED]), only projects
+     *        matching that (e.g., deleted) status will be returned.
+     *
+     * @return A list of [Project] objects matching the specified filter where the given user is member of.
```

Comment:
With the indentation, the references are not correctly resolved, i.e., not seen as references at all.

```suggestion
     * @param statusFilter (optional) Filter to specify which project status the fetched projects should have.
     * By default, [ProjectStatus.PROJECT_STATUS_ACTIVE] is used, which includes both active and active-locked projects.
     * If set to another value (e.g., [ProjectStatus.PROJECT_STATUS_DELETED]), only projects matching that (e.g.,
     * deleted) status will be returned.
     * @return A list of [Project] objects matching the specified filter where the given user is member of.
```

Reply:
Thank you for the hint, I will update it.